tests/vrf/vrf_neigh.j2

```diff
@@ -7,9 +7,9 @@
 {% endif -%}
 {%- endmacro %}
 
-{% for intf, ip_facts in intf_ips.iteritems() -%}
+{% for intf, ip_facts in intf_ips.items() -%}
```

Comment:
Can you please confirm if these changes related to skipping traffic test ? I am not sure if the `iteritems` is not valid anymore ?

Reply:
On a different PR I had a comment about keeping these changes compatible for backporting. So, can this instead be modified to support both Python 2 and 3 using `six`?